app/functions.sh

```diff
@@ -59,8 +78,11 @@ function docker_exec {
     local cmd="${2?missing command}"
     local data=$(printf '{ "AttachStdin": false, "AttachStdout": true, "AttachStderr": true, "Tty":false,"Cmd": %s }' "$cmd")
     exec_id=$(docker_api "/containers/$id/exec" "POST" "$data" | jq -r .Id)
-    if [[ -n "$exec_id" ]]; then
+    if [[ -n $exec_id && ($exec_id != "null") ]]; then
```

Comment:
The parenthesis here are not needed.

I'd prefer `[[ -n "$exec_id" && "$exec_id" != "null" ]]` to stay in line with existing code style.

Reply:
Fixed.